Formula/rbspy.rb

```diff
@@ -0,0 +1,18 @@
+class Rbspy < Formula
+  desc "Sampling profiler for Ruby"
+  homepage "https://github.com/rbspy/rbspy"
+  url "https://github.com/rbspy/rbspy/archive/v0.1.8.tar.gz"
+  sha256 "87f759acd9d660178737b9f24cc07f0113a81a9fffc1604bae2c756d7f4d815b"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build"
```

Comment:
should be `cargo build --release` (to build a release version instead of a debug version)

Reply:
🙇 thanks for the heads up! Addressed this one via 41b3e164